### ISSUE TITLE
Converting NIST db files to .msp

### ISSUE THREAD

**Issue by Stortebecker** [Stortebecker/metabolomics#4]
The commercial NIST db is provided in a closed `Nist db file` format. The freeware `NIST MS search` allows to export spectra to the open `.msp` format. However, conversion is restricted to 1,000 spectra. The whole database consists of approx. 200,000 spectra.

It would be helpful to convert the full db to be able to modify it and use it with open software. @korseby @blankclemens Can we come up with a cunning way to do so? A simple batch script might be sufficient, if the  `NIST MS search` can be controlled via API.

**Comment by Stortebecker** [comment]
Of course, we cannot provide the converted database publicly. But it could be provided for people owning a `NIST db` licence.

**Comment by Stortebecker** [comment]
I guess that this issue could be solved by using either the `MSeasy R Package`, the `NIST MS PepSearch` or the `PyNISTPL`. See links in #9 .

**Comment by Stortebecker** [comment]
Another way would be to use the `NIST MS Search` API directly: [Instructions (PDF)](https://littlemsandsailing.files.wordpress.com/2011/05/detailed_instructions.pdf)

**Comment by gtmddly** [comment]
Hello, I am a college student.，I am doing research on this subject for graduation，and I am interested in your research. Could you please send me a document of MSP format?Here is my email ：1357611445@qq.com                            Thank you for your generosity.

**Comment by hechth** [comment]
Has anyone done some work on this?
